docs/components/landing_pages.rst

```diff
@@ -6,4 +6,51 @@ Landing Pages
 Customizing the Preference Center
 *********************************
 
-.. vale on
\ No newline at end of file
+.. vale on
+
+
+.. vale off
+
+Landing Page Drafts
+============
+
+Creating a draft Landing Page
+-----------------------------
+
+.. vale on
+
+Mautic allows the creation of Landing Page Drafts using the 'Save as Draft' button in the Landing Page editor.
+
+This feature needs turning on by adding the configuration parameter ``page_draft_enabled`` to your ``local.php`` configuration file as detailed below.
+
+.. code:: php
+
+  'page_draft_enabled' => 1
+
+Once turned on, the 'Save as Draft' button appears on the Landing Page edit interface.
+
+.. image:: images/landing_pages/save-draft.png
+  :width: 400
+  :alt: Screenshot showing the 'Save as Draft' button on the Landing Page edit page.
```

Comment:
@RCheesley I'm not sure how to deal with this Vale warning

> Is this referring to a Mautic feature? If so, use 'Landing Page' instead of 'page'.

It should be lower case `p`. Do we have to ignore the Vale check on this line?

Reply:
If you're referring to a Mautic Landing Page then both words are capitalised.